cairo_zero/openzeppelin/ERC20.cairo

```diff
@@ -41,6 +39,13 @@ func totalSupply{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr
     return (totalSupply=totalSupply);
 }
 
+@view
+func total_supply{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() -> (
```

Comment:
Why modifying this file ?
I suppose this is to support the dualVMToken ?
It seems too me the end-to-end tests are using the starknet_token in Cairo folder.
Moreover it corresponds to a specific OZ version v0.6.0 hence any token in this version not having those modification will not be supported by the dualVMToken
Unless I miss something

Reply:
This reflects better the new token standard that has dual casing entrypoints

Reply:
Also - we want the `ERC20.cairo` used in tests & ef-tests to have these snake_case entrypoints to reflect real-world behavior.

Reply:
I see 
So native tokens that are wrapped by dualVMToken will be like this one ?
As for now dualVMToken has been though of to wrap stargate tokens / standard ones.

This needs to be documented somewhere if this changes to ensure what is compatibles with dualVMToken

Reply:
> So native tokens that are wrapped by dualVMToken will be like this one ?

"like this one" as in have both snake_case / camelCase entrypoints - yes
they're a specific version with some more entrypoints related to the bridge logic

> This needs to be documented somewhere if this changes to ensure what is compatibles with dualVMToken

it's as simple as: all tokens adhering to the ERC20 Starknet Token specification are supported

Reply:
more on this here https://docs.openzeppelin.com/contracts-cairo/0.8.0/erc20#:~:text=The%20ERC20%20token%20standard%20is,20%20in%20Cairo%20for%20Starknet.

Reply:
OK I see 
So only  ERC20 Starknet token are supported for DualVmTokens for now as they have snake_case and they need to always send back true for all operations. Any changes in futur (or current version) that do not comply with will make them incompatible.

nit: the comment at the beginning of the file should be updated